src/services/postService.js

```diff
@@ -1,12 +1,33 @@
 const { throwIfError } = require('../utilities/dynamoUtilities');
-const { sendPost } = require("../repository/postDAO");
+const postDAO = require("../repository/postDAO");
+const uuid = require("uuid");
 
 async function createPost(username, text, score, title){
-    const post = await sendPost(username, text, score, title);
-    throwIfError(post);
-    return {message: "Post created successfully"};
+    const post = {class: "post", itemID: uuid.v4(), by: username, desc: text, score, title, replies: []};
+    const data = await postDAO.sendPost(post);
+    throwIfError(data);
+    return post;
+}
+
+async function seePosts(){
+    const posts = await postDAO.scanPosts();
+    throwIfError(posts);
```

Comment:
You should pass the entire response and not just posts, `throwIfError` expects a `$metadata.httpStatusCode` property in the argument you pass.